cmd/context/update-kubeconfig_test.go

```diff
@@ -112,7 +112,8 @@ func Test_updateKubeconfig(t *testing.T) {
 			}
 			defer os.Remove(file)
 
-			if err := ExecuteUpdateKubeconfig(ctx); err != nil {
+			err = ExecuteUpdateKubeconfig(ctx)
```

Comment:
why we cannot use the previous approach?

Reply:
its captured as redundant error check